.travis.yml

```diff
@@ -9,11 +9,7 @@ matrix:
   fast_finish: true
   include:
   - php: 7.2
-    # Disabled since we hit the 50 minute timeout
-#  - php: 7.1
-#    env: DB=sqlite; TYPE=coverage
-#  allow_failures: # We allow this one to fail so fast_finish causes the build to not wait for it
-#  - env: DB=sqlite; TYPE=coverage
+    env: TYPE=coverage
```

Comment:
Thanks for cleaning up the test matrix. Now that we're down to 1 case, could you add a comment outlining possible uses, e.g. PHP 7.3